### PR TITLE
Fix app name and icons web manifest, optimize svg

### PR DESCRIPTION
**Pull request by similicious**
## Proposed change

I noticed that the app name in the manifest has not been updated when the project was forked. So I corrected the name to "Paperless NGX" and added an svg icon, allowing the icon to scale properly on hi res screens. Also I took the liberty to optimize the logo files with svgomg (svgo frontend), reducing the size without changing the content. Lastly I corrected a warning from chrome devtools, saying that the ico size doesn't match the size in the manifest.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/extending.html#back-end-development).
- [x] I have run all `pre-commit` hooks, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/extending.html#code-formatting-with-pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

File: src-ui/src/manifest.webmanifest [hunk 1]
```diff
@@ -5,10 +5,14 @@
...
+    },
+    {
+      "src": "assets/logo-notext.svg",
+      "sizes": "any"
     }
   ],
-  "name": "Paperless NG",
-  "short_name": "Paperless NG",
+  "name": "Paperless NGX",
+  "short_name": "Paperless NGX",
```
Comment (tooomm) [inline src-ui/src/manifest.webmanifest:16]: In most parts the branding is "Paperless-ngx" as far as I'm aware.

Is there an official variant?

Comment (similicious) [reply src-ui/src/manifest.webmanifest:16]: In the title it is "Paperless-ngx"
https://github.com/paperless-ngx/paperless-ngx/blob/666b938550963d136a4f2274cafc0d8d14993761/src-ui/src/index.html#L5

Comment (similicious) [reply src-ui/src/manifest.webmanifest]: I changed the title accordingly. Thank you for the suggestion!

**Comment by tooomm** [comment]
Observations related to the icons (not necessarily this PR) - Not really sure where to drop it, so it leave it here as it is related:
- [logo-dark-notext.svg](https://github.com/similicious/paperless-ngx/blob/fix/web-manifest/src-ui/src/assets/logo-dark-notext.svg) has an oversized background in comparison to [logo-notext.svg](https://github.com/similicious/paperless-ngx/blob/fix/web-manifest/src-ui/src/assets/logo-notext.svg) and [logo-white-notext.svg](https://github.com/similicious/paperless-ngx/blob/fix/web-manifest/src-ui/src/assets/logo-white-notext.svg)
- Same for the [logo.svg](https://github.com/similicious/paperless-ngx/blob/fix/web-manifest/src-ui/src/assets/logo.svg) and [logo-dark.svg](https://github.com/similicious/paperless-ngx/blob/fix/web-manifest/src-ui/src/assets/logo-dark.svg)
  They are also identical? Looking at the naming scheme I would expect the "-dark" one to have a black icon?
- [logo-white-notext.svg](https://github.com/similicious/paperless-ngx/blob/fix/web-manifest/src-ui/src/assets/logo-white-notext.svg) contains `docname="logo-dark-notext.svg"`

**Comment by shamoon** [comment]
Thanks for the PR, I wasn't aware of that tool to reduce the svgs sizes.
@tooomm I dont see an issue here, I maintained the naming from the old logo files according to where theyre used.

**Review by shamoon — APPROVED** [review]

**Pull request merged**